docs/source/index.rst

```diff
@@ -10,6 +10,15 @@ PyTorch documentation
 
 PyTorch is an optimized tensor library for deep learning using GPUs and CPUs.
 
+.. toctree::
+   :glob:
+   :maxdepth: 1
+   :caption: Popular
```

Comment:
What determines if something is put under this category?

Reply:
The idea was to surface content from the latest releases. This isn't a complete list at the moment. 
Also another title in consideration was "Recent" , but I was afraid users would confuse that for recent docs they had looked at.

Reply:
Would a heading like "Quick Links" be better? My problem with "Popular" is that it sounds like we're going to be updating it pretty often based on page views